src/config/backpack/base.php

```diff
@@ -56,11 +56,16 @@
     // (you then need to manually define the routes in your web.php)
     'setup_password_recovery_routes' => true,
 
-    // Set this to true if you would like to enable email verification for your Authenticable model.
-    // Make sure your Authenticable model implements the MustVerifyEmail contract and your
-    // database table contains the email_verified_at column.
+    // Set this to true if you would like to enable email verification for your user model.
+    // Make sure your user model implements the MustVerifyEmail contract and your database 
+    // table contains the `email_verified_at` column. Read the following before enabling:
     // https://backpackforlaravel.com/docs/6.x/base-how-to#enable-email-verification-in-backpack-routes
     'setup_email_verification_routes' => false,
+    
+    // We will automatically add the Verified middleware to the Backpack midleware group
+    // if you disable this you must manually add the verified route middleware to 
+    // the routes you would like to be accessed only by verified users.
+    'add_verified_to_backpack_middleware' => true,
```

Comment:
I don't understand this bit @pxpm , can you please explain? I mean... from this text I understand that Backpack would include the `verified` middleware in its middleware group, so basically in `config('base.middleware_class')`... but

(1) I don't see where that is done in the code;
(2) if it does work that way, wouldn't it be cleaner to tell people to add it themselves? this config is in the same file after all so it's not _that much_ more work to edit the `middleware_class` above; and it's clearer what happens;

Maybe I misunderstand this, though.

Reply:
This middleware is added to the group automatically by the BackpackServiceProvider

It's the default "no config solution". All routes using backpack middleware would be covered by `verified` middleware. Any crud you create/generate will need verification if the feature is enabled, since in `routes/custom.php` the backpack middleware is applied.

I added this new config because you may have special needs, in that case you can disable the automatic middleware and add the verified manually in the routes you need. 

Again, it's like manually registering the aliases. I did it for convenience, it's not something this needs to work. 

We can add another step in the how to if you prefer 🤷

Reply:
Hmm.... ok... let's keep it then, since:
- there's a precedent already... with PasswordRecovery; 
- you make a good point that it's more convenient;
- we do have a way for the user to remove this middleware and add their own;

It's more "magic", but since we some "magic" already with PasswordRecovery, it should be ok.